Gemfile.lock

```diff
@@ -1,19 +1,19 @@
 PATH
   remote: .
   specs:
-    ld-eventsource (1.0.0)
+    ld-eventsource (1.0.1)
       concurrent-ruby (~> 1.0)
       http_tools (~> 0.4.5)
       socketry (~> 0.5.1)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    concurrent-ruby (1.0.5)
+    concurrent-ruby (1.1.6)
     diff-lcs (1.3)
-    hitimes (1.3.0)
+    hitimes (1.3.1)
+    hitimes (1.3.1-java)
```

Comment:
Why is `hitimes` version `1.3.1-java` needed?

Reply:
These are not dependency bumps by me - they're just the result of rerunning `bundle install`, just like if you ran `npm install` you'd see changes in `package-lock.json`. So these are the highest compatible versions that the package manager found at the time. They don't have any effect on someone consuming the library, only on our own build.